Source/WebCore/css/CSSProperties.json

```diff
@@ -8301,7 +8301,7 @@
         "text-decoration-thickness": {
             "codegen-properties": {
                 "converter": "TextDecorationThickness",
-                "parser-grammar": "auto | from-font | <length>",
+                "parser-grammar": "auto | from-font | <length> | <percentage>",
```

Comment:
If we change `<percentage>` to `<length-percentage>`, text-decoration-thickness-valid.html also passes. We need to update it depending on https://github.com/w3c/csswg-drafts/issues/9469.